testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -245,7 +253,7 @@ def test_PDB_hex():
 def test_PDB_metals():
     from MDAnalysis.topology import tables
 
-    u = mda.Universe(PDB_metal, format='PDB')
+    u = mda.Universe(StringIO(PDB_metals), format='PDB')
```

Comment:
Probably doesn't hurt.

That said, CI is failing the same way though here at the moment?

Given your analysis in the matching issue (nothing on our end changed that could sanely cause the failure), and the exposed `pytest` guts in the traceback, I wonder if it may be related to the PyPI release date of `pytest` `6.2.0`, which matches with 24 hours of the first observed failure in an unmerged PR: https://pypi.org/project/pytest/#history

Perhaps worth trying a pin at `pytest==6.1.2` and re-running a few times with that constraint to see if it can consistently pass again?

Reply:
Thanks @tylerjereddy for noticing that, that seems to be doing the trick so far. Do you know if we can re-start just Azure tests without re-running the whole test suite over and over (which is why I've been closing and reopening the PR)?

Reply:
Yeah, I think we can even just restart a single Azure job. If you make an account of some sort on their site I can probably add you as an admin.

Also, I think we need to pin one of the github actions jobs for the exact same reason, looking at the CI just now. Could do that one separately if you want of course.

Reply:
Thanks @tylerjereddy for the approval, I made a Microsoft account at lily@minium.com.au. If you could add me as an admin that would be great for future tests <3

Reply:
![image](https://user-images.githubusercontent.com/7903078/102835333-9ef51180-43b3-11eb-8d54-3140c97627b2.png)

Reply:
Apologies @tylerjereddy, not sure that worked -- clicking the link to join https://dev.azure.com/mdanalysis/ brings me to this error:

<img width="764" alt="Screen Shot 2020-12-23 at 10 32 45 pm" src="https://user-images.githubusercontent.com/31115101/102992358-2dfa5900-456f-11eb-85cc-3b69a1e0c9be.png">

:-(

Reply:
I resent the org invite--should work--it says you've never logged in:

![image](https://user-images.githubusercontent.com/7903078/103020504-bc39f500-4505-11eb-97ae-441eb1d251ee.png)